### DISCUSSION TITLE
iphoneX ios14.4 can't enable this plugin

### DISCUSSION THREAD

**Discussion by L2DO** [remotely-save/remotely-save#111] [Q&A] [answered]
when i install this plugin and enable it，it notices me “failed to load plugin remotely save” Anybody knows what happened？

**Comment by fyears** [comment, marked as answer]
It has been reported by other users that ios 14 could not enable the plugin, while ios 15 is ok.

Unfortunately I do not have ios 14 device so I do not know how to fix this bug…

**Comment by L2DO** [comment]
fyears ***@***.***>于2022年5月14日 周六下午10:49写道：

> It has been reported by other users that ios 14 could not enable the
> plugin, while ios 15 is ok.
>
> Unfortunately I do not have ios 14 device so I do not know how to fix this
> bug…
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/remotely-save/remotely-save/discussions/111#discussioncomment-2751167>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AW6FJLZ2VRHD3IXBJKITBZ3VJ64PPANCNFSM5V5SY3TA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>
Thanks，I will upgrade my ios later
